cli/dcoscli/auth/main.py

```diff
@@ -0,0 +1,109 @@
+import dcoscli
+import docopt
+from dcos import cmds, config, emitting, http, util
+from dcos.errors import DCOSAuthorizationException, DCOSException
+from dcoscli.subcommand import default_command_info, default_doc
+from dcoscli.util import decorate_docopt_usage
+
+from six.moves import urllib
+
+emitter = emitting.FlatEmitter()
+logger = util.get_logger(__name__)
+
+
+def main(argv):
+    try:
+        return _main(argv)
+    except DCOSException as e:
+        emitter.publish(e)
+        return 1
+
+
+@decorate_docopt_usage
+def _main(argv):
+    args = docopt.docopt(
+        default_doc("auth"),
+        argv=argv,
+        version='dcos-auth version {}'.format(dcoscli.version))
+
+    http.silence_requests_warnings()
```

Comment:
Since we are running in the same python env, we can probably just do this in the root main. Okay for now. Can you file a GitHub issue?